README.md

```diff
@@ -10,6 +10,7 @@ Blobfuse2 is stable, and is ***supported by Microsoft*** provided that it is use
 ##  NOTICE
 - We have seen some customer issues around files getting corrupted when `streaming` is used in write mode. Kindly avoid using this feature for write while we investigate and resolve it.
 - You can now use block-cache instead of streaming for both read and write workflows, which offers much better performance compared to streaming. To enable `block-cache` instead of `streaming`, use `--block-cache` in CLI param or `block-cache` as component in config file instead of `streaming`.
+- Following the release of version 2.3.0, blobfuse no longer supports object-ID based OAuth for Managed Identity. Users requiring this feature must utilize CLI-based login and ensure the Azure CLI is installed on their system.

```

Comment:
Or migrate to using app-id based authenication.

Reply:
In my opinion, we should not say that we don't support ObjectID with MI. We can say that it is supported but with the dependency of Azure CLI being installed in their system. If they can't install Azure CLI, they can migrate to app-id based authentication.